content/docs/introduction/comparison.md

```diff
@@ -244,43 +244,39 @@ blackbox probing is sufficient.
 If you want to do whitebox monitoring, or have a dynamic or cloud based
 environment, then Prometheus is a good choice.
 
+From https://prometheus.io/docs/introduction/comparison/
+
 ## Prometheus vs. Sensu
 
-[Sensu](https://sensuapp.org/) is broadly speaking a more modern Nagios.
+[Sensu](https://sensu.io) is a composable monitoring pipeline that can reuse existing Nagios checks.
 
 ### Scope
 
-The same general scope differences as in the case of
-[Nagios](/docs/introduction/comparison/#prometheus-vs-nagios) apply here.
+Sensu checks adhere to the Nagios check specification (https://assets.nagios.com/downloads/nagioscore/docs/nagioscore/3/en/pluginapi.html). Sensu checks operate on two different models: publish and subscribe or standalone. It is possible to silence not just individual checks, but entire subscriptions. 
 
-The primary difference is that Sensu clients [register themselves](https://sensuapp.org/docs/0.27/reference/clients.html#what-is-a-sensu-client),
-and can determine the checks to run either from central or local configuration.
-Sensu does not have a limit on the amount of perfData.
+The primary difference is that Sensu clients [register themselves](https://docs.sensu.io/sensu-core/latest/reference/clients),
+and can determine the checks to run either from central or local configuration. Sensu clients are also able to be automatically deregistered using a number of [plugins](https://github.com/sensu-plugins/sensu-plugins-sensu) and [integrations](https://docs.sensu.io/sensu-enterprise/latest/built-in-handlers/)
```

Comment:
"able to be" is a bit awkward. I'd say:

"Sensu clients can also be automatically deregistered..."

Reply:
:heavy_check_mark: